src/main/java/dev/cammiescorner/arcanuscontinuum/Arcanus.java

```diff
@@ -169,14 +171,16 @@ public void onInitialize(ModContainer mod) {
 					for(BlockPos pos : pointOfInterest.collect(Collectors.toSet())) {
 						BlockState state = world.getBlockState(pos);
 
-						if(state.getBlock() instanceof MagicDoorBlock doorBlock && world.getBlockEntity(pos) instanceof MagicDoorBlockEntity door && message.equals(door.getPassword())) {
-							doorBlock.setOpen(null, world, state, pos, true);
-							player.sendMessage(translate("door", "access_granted").formatted(Formatting.GRAY, Formatting.ITALIC), true);
+						if(state.getBlock() instanceof MagicDoorBlock doorBlock && world.getBlockEntity(pos) instanceof MagicDoorBlockEntity door) {
+							boolean messageIsPassword = message.equals(door.getPassword()) || message.equals(door.getPassword().toLowerCase());
```

Comment:
If you're adding the lowercase version, perhaps it would be best to just ignore casing (by lowercasing both the input and the stored password before comparing).

Also please use the locale-independent version of `toLowerCase()` by passing `Locale.ROOT`.

Reply:
You're right. Just stored the lowercase password and compared to it now